src/phone-input/country-select-dropdown.js

```diff
@@ -22,87 +22,97 @@ import {getOverrides} from '../helpers/overrides.js';
+  overrides: {},
+};
 
-export default CountrySelectDropdown;
+function CountrySelectDropdown(props: CountrySelectDropdownPropsT) {
+  const {
+    children,
+    containerRef,
+    country,
+    overrides,
+    maxDropdownHeight,
+    mapIsoToLabel,
+  } = props;
+  const [Container, containerProps] = getOverrides(
+    overrides.CountrySelectDropdown,
+    DefaultContainer,
+  );
+  const [ListItem, listItemProps] = getOverrides(
+    overrides.CountrySelectDropdownListItem,
+    DefaultListItem,
+  );
+  const [FlagColumn, flagColumnProps] = getOverrides(
+    overrides.CountrySelectDropdownFlagColumn,
+    DefaultFlagColumn,
+  );
+  const [NameColumn, nameColumnProps] = getOverrides(
+    overrides.CountrySelectDropdownNameColumn,
+    DefaultNameColumn,
+  );
+  const [Dialcode, dialcodeProps] = getOverrides(
+    overrides.CountrySelectDropdownDialcodeColumn,
+    DefaultDialcodeColumn,
+  );
+  const scrollIndex = Math.min(
+    children.findIndex(opt => opt.props.item.id === country.id) + 5,
+    children.length - 1,
+  );
+  return (
+    <Container
+      ref={containerRef}
+      $height={maxDropdownHeight}
+      {...containerProps}
+    >
+      <AutoSizer>
+        {({height, width}) => {
+          return (
+            <List
+              role="listbox"
+              height={height}
+              width={width}
+              rowCount={children.length}
+              rowHeight={42}
+              scrollToIndex={scrollIndex}
+              rowRenderer={({index, key, style}) => {
+                // resetMenu and getItemLabel should not end up on native html elements
+                const {resetMenu, getItemLabel, ...rest} = children[
+                  index
+                ].props;
+                return (
+                  <ListItem
+                    key={key}
+                    style={style}
+                    {...rest}
+                    {...listItemProps}
+                    data-e2e="country-select-list-item"
+                    data-iso={children[index].props.item.id}
+                  >
+                    <FlagColumn {...flagColumnProps}>
+                      <StyledFlag iso={children[index].props.item.id} />
+                    </FlagColumn>
+                    <NameColumn {...nameColumnProps}>
+                      {mapIsoToLabel
+                        ? mapIsoToLabel(props.children[index].props.item.id)
+                        : children[index].props.item.label}
+                    </NameColumn>
+                    <Dialcode {...dialcodeProps}>
+                      {children[index].props.item.dialCode}
+                    </Dialcode>
+                  </ListItem>
+                );
+              }}
+            />
+          );
+        }}
+      </AutoSizer>
+    </Container>
+  );
+}
+
+export default React.forwardRef<
+  CountrySelectDropdownPropsT,
+  typeof CountrySelectDropdown,
+>((props, ref) => <CountrySelectDropdown {...props} containerRef={ref} />);
```

Comment:
I usually call this `forwardedRef` to avoid ambiguity and to explicitly call-out that it's an implementation detail required for `React.forwardRef`